### PR TITLE
Update Cloud IDE docs based on recent features

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: astro/cloud-ide/deploy-project.md [hunk 1]
```diff
@@ -47,23 +47,34 @@ The Cloud IDE includes a robust GitHub integration for managing different versio
 
 1. In the Cloud UI, select a Workspace and then click **Cloud IDE**.
 2. Select the project you'd like to deploy.
-3. Click **Configure**.
-4. Enter your GitHub Personal Access Token and click **Update**.
-5. Enter your GitHub repository information.
-6. Click **Update**.
+3. In the **GitHub Repo** menu, click **Configure**.
```
Comment (kwcanuck) [inline astro/cloud-ide/deploy-project.md:50]:
```suggestion
3. In the **GitHub Repo** pane, click **Configure**.
```
image

Also, you should probably request an update to the UI:

image

File: astro/cloud-ide/deploy-project.md [hunk 2]
```diff
@@ -47,23 +47,34 @@ The Cloud IDE includes a robust GitHub integration for managing different versio
...
+4. Configure the following values:
```
Comment (kwcanuck) [inline astro/cloud-ide/deploy-project.md:51]:
```suggestion
4. Complete the following fields: 
```

File: astro/cloud-ide/deploy-project.md [hunk 3]
```diff
@@ -47,23 +47,34 @@ The Cloud IDE includes a robust GitHub integration for managing different versio
...
 1. In the Cloud UI, select a Workspace and then click **Cloud IDE**.
 2. Select the project you'd like to deploy.
-3. Click **Configure**.
-4. Enter your GitHub Personal Access Token and click **Update**.
-5. Enter your GitHub repository information.
-6. Click **Update**.
+3. In the **GitHub Repo** menu, click **Configure**.
+4. Configure the following values: 
+
+    - **PERSONAL ACCESS TOKEN**: Your personal access token.
```
Comment (kwcanuck) [inline astro/cloud-ide/deploy-project.md:53]:
```suggestion
    - **Personal Access Token**: Enter your personal access token. If you don't have an access token, follow the instructions to generate one.
```

File: astro/cloud-ide/deploy-project.md [hunk 4]
```diff
@@ -47,23 +47,34 @@ The Cloud IDE includes a robust GitHub integration for managing different versio
...
+    - **REPOSITORY**: Your GitHub repository, formatted as `<owner>/<repository>`
```
Comment (kwcanuck) [inline astro/cloud-ide/deploy-project.md:54]:
```suggestion
    - **Repository**: Enter the name of your GitHub repository. The format is `<owner>/<repository>`.
```

File: astro/cloud-ide/deploy-project.md [hunk 5]
```diff
@@ -47,23 +47,34 @@ The Cloud IDE includes a robust GitHub integration for managing different versio
...
+    - **DEFAULT BRANCH**: The branch you want to commit your project to by default.
```
Comment (kwcanuck) [inline astro/cloud-ide/deploy-project.md:55]:
```suggestion
    - **Default Branch**: Select the branch you want to commit your project to.
```

File: astro/cloud-ide/deploy-project.md [hunk 6]
```diff
@@ -47,23 +47,34 @@ The Cloud IDE includes a robust GitHub integration for managing different versio
...
+    - **CLONE REPO**: Turn on **Clone GitHub repo during cell execution** if you want the Cloud IDE to have access to your GitHub repo files when executing an individual cell. Turn on this feature if you have cells that depend on helper files, such as helper functions in `include`.
```
Comment (kwcanuck) [inline astro/cloud-ide/deploy-project.md:56]:
```suggestion
    - **Clone GitHub repo during cell execution**:  Click the toggle to allow the Cloud IDE to access your GitHub repository files when executing an individual cell. Turn on this feature if you have cells that depend on helper files, such as helper functions in `include`.
```

Comment (kwcanuck) [reply astro/cloud-ide/deploy-project.md:56]: What about **Disable auto sync in favor of manual sync**?

image

File: astro/cloud-ide/deploy-project.md [hunk 7]
```diff
@@ -47,23 +47,34 @@ The Cloud IDE includes a robust GitHub integration for managing different versio
...
+
+6. Click **Configure**.
 
 After you configure a GitHub repository, the **Configure** button in the Cloud IDE updates to **Commit**. You can now commit your changes to your repository.
 
 1. Click **Commit**.
-2. Click **New Branch**, then choose from either an existing branch or create a new branch.
+2. Select a **BRANCH TYPE** and **BRANCH NAME** for your commit.
```
Comment (kwcanuck) [inline astro/cloud-ide/deploy-project.md:63]:
```suggestion
2. Select a branch type in the **Branch Type** list and then enter a name for the branch in the **Branch Name** field.
```

File: astro/cloud-ide/deploy-project.md [hunk 8]
```diff
@@ -47,23 +47,34 @@ The Cloud IDE includes a robust GitHub integration for managing different versio
...
 3. In the **COMMIT MESSAGE** field, enter a commit message. This will be the commit message for committing your changes from the Astro Cloud IDE to GitHub.
```
Comment (kwcanuck) [inline astro/cloud-ide/deploy-project.md:64]:
```suggestion
3. In the **Commit Message** field, enter a commit message. This is the commit message for changes committed  from the Astro Cloud IDE to GitHub.
```

File: astro/cloud-ide/deploy-project.md [hunk 9]
```diff
@@ -47,23 +47,34 @@ The Cloud IDE includes a robust GitHub integration for managing different versio
...
+
+:::
+
 On the first GitHub commit of your project, the Astro Cloud IDE automatically pushes a Github action to your repository that includes steps for deploying to Astro. You can then configure your GitHub repository to push your Astro Cloud IDE project to a Deployment when you commit to specific branches. 
 
-You can commit changes from the Astro Cloud IDE to your GitHub repository without configuring Deployments in GitHub. However, there is currently no way to disable the GitHub Action that the Astro Cloud IDE pushes to your repository. If you don't complete the following steps, the GitHub Action fails and you can't deploy your changes to Astro, but the rest of your commit processes are successful.
+You can commit changes from the Astro Cloud IDE to your GitHub repository without configuring Deployments in GitHub. However, there is currently no way to disable the GitHub action that the Astro Cloud IDE pushes to your repository. If you don't complete the following steps, the GitHub action fails and you can't deploy your changes to Astro, but the rest of your commit processes are successful.
```
Comment (kwcanuck) [inline astro/cloud-ide/deploy-project.md:77]:
```suggestion
You can commit changes from the Astro Cloud IDE to your GitHub repository without configuring Deployments in GitHub. However, you can't disable the GitHub action that the Astro Cloud IDE pushes to your repository. If you don't complete the following steps, the GitHub action fails, you can't deploy your changes to Astro, but the rest of your commit processes are successful.
```

**Review by kwcanuck — APPROVED** [review]

File: astro/cloud-ide/deploy-project.md [hunk 10]
```diff
@@ -47,23 +47,35 @@ The Cloud IDE includes a robust GitHub integration for managing different versio
...
 4. Click **Commit**.
 
 ## Deploy a project from GitHub to Astro
 
+:::info
+
+To deploy your project to Astro with the default GitHub action provided by the Cloud IDE, you must enable DAG-only deploys on your Deployment. See [Deploy DAGs only](deploy-code.md#deploy-dags-only).
+
+:::
+
```
Comment (jlaneve) [inline astro/cloud-ide/deploy-project.md:75]: We enable this by default with our CI/CD. Not sure that warrants changing these docs, just wanted to let you know

**Review by jlaneve — APPROVED** [review]
Left one minor comment, otherwise looks good!

File: astro/cloud-ide/deploy-project.md (hunk 10 above)
Comment (jwitz) [inline astro/cloud-ide/deploy-project.md:75]:
```suggestion
```

File: astro/cloud-ide/deploy-project.md [hunk 11]
```diff
@@ -47,23 +47,35 @@ The Cloud IDE includes a robust GitHub integration for managing different versio
...
 On the first GitHub commit of your project, the Astro Cloud IDE automatically pushes a Github action to your repository that includes steps for deploying to Astro. You can then configure your GitHub repository to push your Astro Cloud IDE project to a Deployment when you commit to specific branches. 
 
-You can commit changes from the Astro Cloud IDE to your GitHub repository without configuring Deployments in GitHub. However, there is currently no way to disable the GitHub Action that the Astro Cloud IDE pushes to your repository. If you don't complete the following steps, the GitHub Action fails and you can't deploy your changes to Astro, but the rest of your commit processes are successful.
+You can commit changes from the Astro Cloud IDE to your GitHub repository without configuring Deployments in GitHub. However, you can't disable the GitHub action that the Astro Cloud IDE pushes to your repository. If you don't complete the following steps, the GitHub action fails, you can't deploy your changes to Astro, but the rest of your commit processes are successful.
```
Comment (jwitz) [inline astro/cloud-ide/deploy-project.md:79]:
```suggestion

:::caution

The GitHub action first runs, it automatically enables DAG-only deploys on your Deployment, which is a Public Preview feature that changes how DAGs and configuration changes are pushed to the Deployment. See [Deploy DAGs only](deploy-code.md#deploy-dags-only).

:::

```

File: astro/cloud-ide/deploy-project.md [hunk 12]
```diff
@@ -47,23 +47,35 @@ The Cloud IDE includes a robust GitHub integration for managing different versio
...
+
+:::caution
+
+The GitHub action first runs, it automatically enables DAG-only deploys on your Deployment, which is a Public Preview feature that changes how DAGs and configuration changes are pushed to the Deployment. See [Deploy DAGs only](deploy-code.md#deploy-dags-only).
```
Comment (jwitz) [inline astro/cloud-ide/deploy-project.md:76]:
```suggestion
When the GitHub action first runs, it automatically enables DAG-only deploys on your Deployment, which is a Public Preview feature that changes how DAGs and configuration changes are pushed to the Deployment. See [Deploy DAGs only](deploy-code.md#deploy-dags-only).
```

**Pull request merged**